cpp/include/cudf/functions.h

```diff
@@ -820,114 +2534,345 @@ gdf_error gdf_max(gdf_column *col, void *dev_result, gdf_size_type dev_result_si
-                           gdf_column* col_agg,          //column to aggregate on with 0 null_count otherwise GDF_VALIDITY_UNSUPPORTED is returned
-                           gdf_column* out_col_indices,  //if not null return indices of re-ordered rows
-                           gdf_column** out_col_values,  //if not null return the grouped-by columns
-                                                         //(multi-gather based on indices, which are needed anyway)
-                           gdf_column* out_col_agg,      //aggregation result
-                           gdf_context* ctxt);            //struct with additional info: bool is_sorted, flag_sort_or_hash, bool flag_count_distinct
-
-gdf_error gdf_group_by_count(int ncols,                    // # columns
-                             gdf_column** cols,            //input cols with 0 null_count otherwise GDF_VALIDITY_UNSUPPORTED is returned
-                             gdf_column* col_agg,          //column to aggregate on with 0 null_count otherwise GDF_VALIDITY_UNSUPPORTED is returned
-                             gdf_column* out_col_indices,  //if not null return indices of re-ordered rows
-                             gdf_column** out_col_values,  //if not null return the grouped-by columns
-                                                         //(multi-gather based on indices, which are needed anyway)
-                             gdf_column* out_col_agg,      //aggregation result
-                             gdf_context* ctxt);            //struct with additional info: bool is_sorted, flag_sort_or_hash, bool flag_count_distinct
-
-gdf_error gdf_quantile_exact(	gdf_column*         col_in,       //input column with 0 null_count otherwise GDF_VALIDITY_UNSUPPORTED is returned
-                                gdf_quantile_method prec,         //precision: type of quantile method calculation
-                                double              q,            //requested quantile in [0,1]
-                                void*               t_erased_res, //result; for <exact> should probably be double*; it's void* because
-                                                                  //(1) for uniformity of interface with <approx>;
-                                                                  //(2) for possible types bigger than double, in the future;
-                                gdf_context*        ctxt);        //context info
-
-gdf_error gdf_quantile_aprrox(	gdf_column*  col_in,       //input column with 0 null_count otherwise GDF_VALIDITY_UNSUPPORTED is returned
-                                double       q,            //requested quantile in [0,1]
-                                void*        t_erased_res, //type-erased result of same type as column;
-                                gdf_context* ctxt);        //context info
+/* --------------------------------------------------------------------------*/
+/**
+ * @brief  Performs SQL like WHERE (Filtering)
```

Comment:
I think this is a good brief comment, but documentation should also include (after the brief) detailed explanation of the functions semantics, expected use, etc. Perhaps that is a task for a further PR. An issue should be filed if so.

Reply:
I will create an issue in that case. We did not create this function, nor do we use it, so we would not be the best for providing a detailed explanation of the functions semantics, expected use, etc